### DISCUSSION TITLE
"Void" category is not being ignored

### DISCUSSION THREAD

**Discussion by kadattack** [bowenc0221/panoptic-deeplab#85] [Q&A]
Does Panoptic-Deeplab have implemented the ability to ignore certain categories?
The COCO dataset i am using has an "ignore" category using ID number 4.
What is the proper way to set the "void" or "ignore" category so that model will ignore it while learning?

I've tried setting the Metadata
ignore_label=4
as shown here:
https://detectron2.readthedocs.io/en/latest/_modules/detectron2/data/datasets/coco_panoptic.html?highlight=ignore_label

And i've also tried setting the config file
SEM_SEG_HEAD.IGNORE_VALUE : 4
as shown here:
https://detectron2.readthedocs.io/en/latest/modules/config.html?highlight=ignore_value

None of these seem to work for me.
The model still learners the "ignore" value when visualising
Example of blue color ignore:
[https://i.imgur.com/e6iA4lN.jpeg](https://i.imgur.com/e6iA4lN.jpeg)

**Comment by bowenc0221** [comment]
I guess `cfg.MODEL.SEM_SEG_HEAD.IGNORE_VALUE` is what you are looking for?

**Reply by kadattack** [reply]
Yes i tried this.
It doesn't seem to work, that's why I thought it was maybe Panoptic-Deeplab related on why it doesn't work.
Guess not.
I just removed the "Ignore" class from the COCO dataset now and trained without it, to fix this.

**Reply by bowenc0221** [reply]
I see what you mean. Do you want to not show the ignore class in your visualization? In this case, yes, you should not but ignore class in the dataset.
